core/src/main/java/com/expediagroup/streamplatform/streamregistry/core/services/ConsumerBindingService.java

```diff
@@ -23,94 +25,81 @@
 
-import org.springframework.data.domain.Example;
 import org.springframework.stereotype.Component;
 
-import com.expediagroup.streamplatform.streamregistry.DataToModel;
-import com.expediagroup.streamplatform.streamregistry.ModelToData;
 import com.expediagroup.streamplatform.streamregistry.core.events.EventType;
 import com.expediagroup.streamplatform.streamregistry.core.events.NotificationEventEmitter;
 import com.expediagroup.streamplatform.streamregistry.core.handlers.HandlerService;
-import com.expediagroup.streamplatform.streamregistry.core.repositories.ConsumerBindingRepository;
 import com.expediagroup.streamplatform.streamregistry.core.validators.ConsumerBindingValidator;
 import com.expediagroup.streamplatform.streamregistry.core.validators.ValidationException;
-import com.expediagroup.streamplatform.streamregistry.data.ConsumerBindingData;
-import com.expediagroup.streamplatform.streamregistry.data.keys.ConsumerBindingDataKey;
 import com.expediagroup.streamplatform.streamregistry.model.ConsumerBinding;
 import com.expediagroup.streamplatform.streamregistry.model.keys.ConsumerBindingKey;
 import com.expediagroup.streamplatform.streamregistry.model.keys.ConsumerKey;
+import com.expediagroup.streamplatform.streamregistry.repository.ConsumerBindingRepository;
 
 @Component
 @RequiredArgsConstructor
 public class ConsumerBindingService {
 
-  private final DataToModel dataToModel;
-  private final ModelToData modelToData;
   private final HandlerService handlerService;
   private final ConsumerBindingValidator consumerBindingValidator;
   private final ConsumerBindingRepository consumerBindingRepository;
   private final NotificationEventEmitter<ConsumerBinding> consumerBindingServiceEventEmitter;
 
   public Optional<ConsumerBinding> create(ConsumerBinding consumerBinding) throws ValidationException {
-    consumerBindingValidator.validateForCreate(consumerBinding);
-    ConsumerBindingData consumerBindingData = modelToData.convertToData(consumerBinding);
-    if (consumerBindingRepository.findById(consumerBindingData.getKey()).isPresent()) {
+    if (read(consumerBinding.getKey()).isPresent()) {
       throw new ValidationException("Can't create because it already exists");
     }
-    consumerBindingData.setSpecification(modelToData.convertToData(handlerService.handleInsert(consumerBinding)));
-    ConsumerBinding out = dataToModel.convertToModel(consumerBindingRepository.save(consumerBindingData));
-    consumerBindingServiceEventEmitter.emitEventOnProcessedEntity(EventType.CREATE, out);
-    return Optional.ofNullable(out);
-  }
-
-  public Optional<ConsumerBinding> read(ConsumerBindingKey key) {
-    var data = consumerBindingRepository.findById(modelToData.convertToData(key));
-    return data.map(dataToModel::convertToModel);
+    consumerBindingValidator.validateForCreate(consumerBinding);
+    consumerBinding.setSpecification(handlerService.handleInsert(consumerBinding));
+    return save(consumerBinding, CREATE);
```

Comment:
You have removed the notifiers, I assume this was deliberate?
Has the notifier code also been removed?